### PR TITLE
Additional Extension Points for Params and extending Request examples

### PR DESCRIPTION
**Pull request by ue71603**
Many Extensions affect the parameters of a request. This PR adds the missing Extension points.
For the Location information request this had to be done with a different name as otherwise an ambiguity occurs (as PlaceParamStructure can be completly empty).

### CODE REVIEW

File: OJP/OJP_Locations.xsd [hunk 1]
```diff
@@ -84,6 +84,7 @@
 			<xs:group ref="PlaceDataFilterGroup"/>
 			<xs:group ref="PlacePolicyGroup"/>
 			<xs:group ref="PlaceSortingGroup"/>
+			<xs:element name="ParamExtension" type="xs:anyType" minOccurs="0"/>
```
Comment (skinkie) [inline OJP/OJP_Locations.xsd:87]: Is there a reason you name it ParamExtension and not just Extension here?
```suggestion
			<xs:element name="Extension" type="xs:anyType" minOccurs="0"/>
```

**Comment by ue71603** [comment]
check for Extension in LIR/Restrictions

**Review by skinkie — APPROVED** [review]

**Review by sgrossberndt — APPROVED** [review]

**Comment by sgrossberndt** [comment]
@ue71603  After merging I had to adapt `TR_01_request_Bern_Luzern_10_results_optimisation_fastest.xml` as it was in the wrong order: 

https://github.com/VDVde/OJP/pull/369/files#diff-bd26ebd43d28c5987af0174c9ffc584b092652f2e94e16e1ca86927a421b425b

`BikeTransport` must be before `NumberOfResults`

I wonder why this was not caught by the previous CI run: https://github.com/VDVde/OJP/actions/runs/5271618917/jobs/9532760995#step:6:52

**Comment by ue71603** [comment]
I thought it might be the folders, but they are set correctly and all examples should have been validated.

**Pull request merged**